packages/cli/src/zostso/issue/command/Command.handler.ts

```diff
@@ -36,6 +37,15 @@ export default class Handler extends ZosTsoBaseHandler {
                 addressSpaceOptions: this.mTsoStart
             }
         );
+        const defProc = "IZUFPROC";
+        const logProc = params.arguments.logonProcedure;
+        if (params.arguments.logonProcedure && params.arguments.logonProcedure !== defProc && params.arguments.suppressStartupMessages) {
+            this.console.error(
+                chalk.yellow(
+                    "Warning: The logon procedure specified is not used when issuing a TSO command with the suppressStartupMessages (--ssm) option set to true."
+                )
+            );
+        }
```

Comment:
We should either use `logProc` or remove it as its unused in this code.

```suggestion
        const logProc = params.arguments.logonProcedure;
        if (logProc && logProc !== defProc && params.arguments.suppressStartupMessages) {
            this.console.error(
                chalk.yellow(
                    "Warning: The logon procedure specified is not used when issuing a TSO command with the suppressStartupMessages (--ssm) option set to true."
                )
            );
        }
```